java.jdbc-ballerina/Module.md

```diff
@@ -0,0 +1,96 @@
+command, you should be able to run the resultant by executing the `ballerina run` command.
+
+E.g., The `Ballerina.toml` content.
+Change the path to the JDBC driver appropriately.
+
+```toml
+[package]
+org = "sample"
+name = "jdbc"
+version= "0.1.0"
+
+[[platform.java11.dependency]]
+artifactId = "h2"
+version = "1.4.200"
+path = "/path/to/com.h2database.h2-1.4.200.jar"
+groupId = "com.h2database"
+``` 
+
+### Client
+To access a database, you must first create a
+[jdbc:Client](https://ballerina.io/learn/api-docs/ballerina/#/java.jdbc/clients/Client) object.
+The examples for creating a JDBC client can be found below.
+
+#### Creating a Client
+This example shows the different ways of creating the `jdbc:Client`. The client can be created by passing
+the JDBC URL, which is a mandatory property and all other fields are optional.
+
+The `dbClient1` receives only the database URL and the `dbClient2` receives the username and password in addition to the URL.
+If the properties are passed in the same order as it is defined in the `jdbc:Client`, you can pass it
+without named params.
+
+The `dbClient3` uses the named params to pass all the attributes and provides the `options` property in the type of
+[jdbc:Options](https://ballerina.io/learn/api-docs/ballerina/#/java.jdbc/records/Options)
+and also uses the unshared connection pool in the type of
+[sql:ConnectionPool](https://ballerina.io/learn/api-docs/ballerina/#/sql/records/ConnectionPool).
+For more information about connection pooling, see [SQL Module](https://ballerina.io/learn/api-docs/ballerina/#/sql).
+
+The `dbClient4` receives some custom properties within the
+[jdbc:Options](https://ballerina.io/learn/api-docs/ballerina/#/java.jdbc/records/Options)
+and those properties will be used by the defined `datasourceName`.
+As per the provided example, the `org.h2.jdbcx.JdbcDataSource` datasource  will be configured with a `loginTimeout`
+of `2000` milliseconds.
+
+```ballerina
+jdbc:Client|sql:Error dbClient1 = new ("jdbc:h2:~/path/to/database");
+jdbc:Client|sql:Error dbClient2 = new ("jdbc:h2:~/path/to/database", 
+                            "root", "root");
+jdbc:Client|sql:Error dbClient3 = new (url =  "jdbc:h2:~/path/to/database",
+                             user = "root", password = "root",
+                             options = {
+                                 datasourceName: "org.h2.jdbcx.JdbcDataSource"
+                             },
+                             connectionPool = {
+                                 maxOpenConnections: 5
+                             });
+jdbc:Client|sql:Error dbClient4 = new (url =  "jdbc:h2:~/path/to/database", 
+                             user = "root", password = "root",
+                             options = {
+                                datasourceName: "org.h2.jdbcx.JdbcDataSource", 
+                                properties: {"loginTimeout": "2000"}
+                             });                          
+```
```

Comment:
Shall we move these code snippets to align with the description? So don't need to name differently. WDYT?

Reply:
changed in https://github.com/ballerina-platform/module-ballerinax-java.jdbc/pull/160/commits/1f1371b1af991a468da03c03512615a6889583b2